process_onestop_sr_report/preprocessing.py

```diff
@@ -223,6 +218,9 @@ class ArgsParser(Tap):
     qas_prolific_distribution_path: Path | None = None
     add_question_in_prompt: bool = False  # whether to add the question in the prompt
     mode: Mode = Mode.IA  # whether to use interest area or fixation data
+    device: str = "cuda" if torch.cuda.is_available() else "cpu"
+    
+    hf_access_token: str = None
```

Comment:
as this is the entry point to the code, maybe mention that this is necessary for a certain surprisal model? link how to get it?